app/pages/page-list.ts

```diff
@@ -24,5 +25,10 @@ export var PAGE_LIST: Array<Page> = [
         title: "Amigo Requests",
         component: AmigoRequestsPage,
         icon: "person-add"
+    },
+    {
+        title: "My Medications",
+        component: MedListPage,
+        icon: "medkit"
```

Comment:
yeah thats a placeholder icon, I wanted to discuss what icon we should go with. Maybe a new one?